tests/robustness/snapmeta/kopia_snapshotter.go

```diff
@@ -200,7 +201,12 @@ func (ks *KopiaSnapshotter) GetRepositoryStatus() (cli.RepositoryStatus, error)
 // UpgradeRepository upgrades the given kopia repository
 // from current format version to latest stable format version.
 func (ks *KopiaSnapshotter) UpgradeRepository() error {
-	_, _, err := ks.snap.Run("repository", "set-parameters", "--upgrade")
+	os.Setenv("KOPIA_UPGRADE_LOCK_ENABLED", "1")
```

Comment:
Let's make sure this does not clobber future tests or any other runtime. Is there a way to have a "local env session"?

Reply:
On way to do this is
``` go
defer os.unsetenv(...)
os.Setenv(...)
ks.snap.run(...)
```

or perhaps to get and set this variable outside this function using `testing.T.Setenv()`. See [this page](https://www.reddit.com/r/golang/comments/ar5z3i/how_to_set_env_variables_while_unit_testing/) for inspiration.

Reply:
Added setenv to cleanup function in the main test. Testing.T.Setenv() is not available in main test.